eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -283,11 +283,12 @@ stages:
                 MarkPackageAsShipped: true
                 ArtifactName: packages-signed
                 SourceRootPath: $(Pipeline.Workspace)/azure-sdk-for-java
-            - template: /eng/common/pipelines/templates/steps/mark-release-completion.yml
-              parameters:
-                ConfigFileDir: $(Pipeline.Workspace)/packages-signed/PackageInfo
-                SourceRootPath: $(Pipeline.Workspace)/azure-sdk-for-java
-                PackageArtifactName: '*'
+            - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
I don't think this will work for java's batch release. We generally cannot loop over the artifact parameters but instead depend on the PackageInfos

Reply:
@weshaggard PackageInfo file is still present for all packages even if I uncheck a package from release list when queueing the pipeline. Artifact list is updated based on the selection at run time and it is used by create release tag step.

Here is the run that triggered release step only for the package I chose.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=5258048&view=logs&j=fbf07650-9b13-5e87-a054-01a63335f5b0&t=57de9a82-9dcf-59ee-d37a-bbe24e586330&l=54